### PR TITLE
fix(script): replace OS not ARCH variable

### PR DESCRIPTION
**Pull request by talalashraf**
Allow script to fail when binary not downloaded
Override arch with amd64 when its x86_64

### CODE REVIEW

File: join/join-testnet-with-binaries.sh [hunk 1]
```diff
@@ -53,8 +53,8 @@ addPeers() {
 }
 
 # override OS with amd64 for x86 arch
-if [ "x86_64" =  "$OS" ]; then
-  OS=amd64
+if [ "x86_64" =  "$ARCH" ]; then
```
Comment (cgorenflo) [inline join/join-testnet-with-binaries.sh:56]: fix the comment

File: join/join-testnet-with-binaries.sh [hunk 2]
```diff
@@ -90,20 +90,20 @@ mkdir -p "$CORE_DIRECTORY"
 CONFIG_DIRECTORY="${CORE_DIRECTORY}/config"
 mkdir -p "$CONFIG_DIRECTORY"
 
-AXELARD_BINARY="axelard-${OS}-${ARCH}-${AXELAR_CORE_VERSION}"
+AXELARD_BINARY="axelard1-${OS}-${ARCH}-${AXELAR_CORE_VERSION}"
```
Comment (cgorenflo) [inline join/join-testnet-with-binaries.sh:93]: why axelard1?

File: join/launch-validator-with-binaries.sh [hunk 3]
```diff
@@ -67,6 +67,11 @@ if [ -z "$TOFND_VERSION" ]; then
   exit 1
 fi
 
+# override OS with amd64 for x86 arch
```
Comment (cgorenflo) [inline join/launch-validator-with-binaries.sh:70]: fix comment

**Review by cgorenflo — CHANGES_REQUESTED** [review]

File: join/join-testnet-with-binaries.sh (hunk 1 above)
Comment (talalashraf) [reply join/join-testnet-with-binaries.sh:56]: comment is correct, we are overriding os with amd64 when its x86

Comment (talalashraf) [reply join/join-testnet-with-binaries.sh:56]: oh nvm

**Review by cgorenflo — APPROVED** [review]

**Pull request merged**